docker/docker_client_test.go

```diff
@@ -104,11 +104,15 @@ func TestGetAuth(t *testing.T) {
 		os.Setenv(homedir.Key(), origHomeDir)
 	}()
 
-	configDir := filepath.Join(tmpDir, ".docker")
-	if err := os.Mkdir(configDir, 0750); err != nil {
+	configDir := filepath.Join(tmpDir, ".config")
+	if err := os.Mkdir(configDir, 0700); err != nil {
 		t.Fatal(err)
 	}
-	configPath := filepath.Join(configDir, "config.json")
+	configDir = filepath.Join(tmpDir, "containers")
+	if err := os.Mkdir(configDir, 0700); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Doesn’t this create `$tmpDir/.config` and `$tmpDir/containers` instead of `$tmpDir/.config/containers`? Also could use `MkdirAll`.

Reply:
fixed.